cantabular-import/analysis/full-import-export/main.go

```diff
@@ -252,13 +254,13 @@ func main() {
 		if instanceFromAPI.Version.State == "edition-confirmed" {
 			// fmt.Printf("\ninstanceFromAPI: %v\n", instanceFromAPI)
 			// spew.Dump(instanceFromAPI)
-			fmt.Printf("Got 'edition-confirmed' after: %d milliseconds\n", 100*(201-attempts))
+			fmt.Printf("Got 'edition-confirmed' after: %d milliseconds\n", StateVerifyPeriod*(MaxAttempts+1-attempts))
 			break
 		}
 		attempts--
 	}
 	if attempts == 0 {
-		fmt.Printf("failed to see 'edition-confirmed' after 20 seconds\n")
+		fmt.Printf("failed to see 'edition-confirmed' after %d seconds\n", MaxAttempts/10)
```

Comment:
sorry to be a bit picky, but if we change the period then this message would be wrong. 
Could you replace this with `MaxAttempts*StateVerifyPeriod/1000` please? (assuming the StateVerifyPeriod is always given in [ms])